recipe/meta.yaml

```diff
@@ -1,24 +1,19 @@
-{% set name = "TileDB" %}
 {% set version = "1.6.2" %}
-{% set sha256 = "001e49f4a9408f0e9064628ddcde201a5f76cf7a87c5bb17799c45e648001393" %}
 
 package:
-  name: {{ name|lower }}
+  name: tiledb
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/TileDB-Inc/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  #git_url: https://github.com/TileDB-Inc/TileDB
```

Comment:
Did the re-render take this out?

Reply:
I did. There is no need to keep that commented in there and the recipe is more readable without it.